packages/front-end/pages/experiment/[eid].tsx

```diff
@@ -754,6 +775,34 @@ const ExperimentPage = (): ReactElement => {
                   </div>
                 </RightRailSection>
               )}
+              <hr />
+              <RightRailSection title="Watcher List" key="WatcherList">
+                <tr
+                  onClick={async () => {
+                    setWatcherListOpen(!watcherListOpen);
+                  }}
+                >
+                  <td>
+                    {" "}
+                    <RightRailSectionGroup
+                      title={`Watchers - ${allUserIds?.length}`}
+                    >
+                      {" "}
+                    </RightRailSectionGroup>
+                  </td>
+                  <td style={{ width: 30 }}>
+                    {watcherListOpen ? <FaAngleUp /> : <FaAngleDown />}
+                  </td>
+                </tr>
+                {watcherListOpen &&
+                  allUserIds.map((id) => (
+                    <tr key={id}>
+                      <td colSpan={4} className="bg-light p-0.5">
+                        {users.has(id) ? users.get(id).name : id}
+                      </td>
+                    </tr>
+                  ))}
```

Comment:
`<tr>` needs to be inside of a `<table>` tag.  But I don't think you need to use tables here at all.  Just a simple list is fine.  Also, I might just always have it expanded instead of letting the user toggle it.

The reason to make it toggled is if we wanted to lazy load the list of users.  If that was an expensive API call, we could wait until someone expands it to fetch the list of users and show it.  So maybe for now, just get rid of the open/close toggle and we can add it later with lazy loading if it becomes a performance issue.